projects/storefrontlib/shared/components/list-navigation/pagination/pagination.component.ts

```diff
@@ -68,7 +68,7 @@ export class PaginationComponent {
    * @param type PaginationItemType
    * @returns string
    */
-  getAriaLabel(label: string, type: PaginationItemType): string {
+  getAriaLabel(label?: string, type?: PaginationItemType): string {
```

Comment:
I think this is a okay case to use optional instead of undefined (from the discussion we had before in the other PR)